draft-ietf-ppm-dap.md

```diff
@@ -1239,30 +1247,27 @@ struct {
 } PrepareStep;
 
 struct {
+  u16 round;
   PrepareStep prepare_steps<1..2^32-1>;
 } AggregateInitializeResp;
 ~~~
 
-The message is a sequence of PrepareStep values, the order of which matches that
-of the ReportShare values in `AggregateInitializeReq.report_shares`. Each report
-that was marked as invalid is assigned the PrepareStepResult `failed`.
-Otherwise, the PrepareStep is either marked as continued with the output
-`prep_msg`, or is marked as finished if the VDAF preparation process is finished
-for the report share.
+The message consists of:
+
+* `round` is the VDAF preparation round that the job is on, which is 0 when the
+  Helper responds to the Leader's `AggregationJobInitReq`.
```

Comment:
```suggestion
  Helper responds to the Leader's `AggregationInititalizationReq`.
```

Reply:
This is confusing, but `AggregationJobInitReq` is correct. The diff looked funny when you reviewed because I forgot to rebase on `main` after merging #398.